qa/tasks/cephfs/kernel_mount.py

```diff
@@ -217,15 +229,52 @@ def get_global_id(self):
 
         assert self.mounted
 
-        mds_sessions = self._read_debug_file("mds_sessions")
+        mds_sessions = self.read_debug_file("mds_sessions")
+        assert mds_sessions 
+
         lines = mds_sessions.split("\n")
         return int(lines[0].split()[1])
 
+    @property
+    def _global_addr(self):
+        if self.addr is not None:
+            return self.addr
+
+        status = self.read_debug_file("status")
+        if status is None:
+            return None
+
+        self.addr = status.split()[2].split(')')[1]
```

Comment:
Can you note the syntax of the line so future readers of the code know what the input looks like.

Reply:
Sure.

Reply:
Sure, will add it as one comments and will past it here:
 
The libcephfs will be:

```
# ./bin/ceph --admin-daemon /tmp/ceph-asok.n5UeBU/client.admin.256955.asok status
*** DEVELOPER MODE: setting PATH, PYTHONPATH and LD_LIBRARY_PATH ***
{

...
    "inst_str": "client.26061 10.72.47.117:0/2102877515",
    "addr_str": "10.72.47.117:0/2102877515",

...
}
```

The kclient will be:

```
# cat /sys/kernel/debug/ceph/cca096e8-7e84-4838-a396-39bb94001376.client4297/status 
instance: client.4297 (0)10.72.47.117:0/1148470933
blocklisted: false
```

Reply:
The addr and inst output will be like:

`2020-11-17 09:29:53,300.300 INFO:tasks.cephfs.test_XXX: addr = 10.72.47.117:0/459239585, inst = client29604 10.72.47.117:0/459239585`